src/modelinstance.cpp

```diff
@@ -1302,14 +1392,123 @@ Status ModelInstance::infer(const RequestType* requestProto,
             SPDLOG_DEBUG("Used device: {}", device);
 
     status = requestProcessor->release();
+    SPDLOG_DEBUG("Calling user provided callback");  // TODO check if this shows
+    // handleCallback(requestProto, responseProto);
     return status;
 }
+
+#pragma GCC diagnostic pop
+template <typename RequestType, typename ResponseType>
+Status ModelInstance::inferAsync(const RequestType* requestProto,
+    std::unique_ptr<ModelInstanceUnloadGuard>& modelUnloadGuardPtr) {
+    OVMS_PROFILE_FUNCTION();
+    Timer<TIMER_END> timer;
+    using std::chrono::microseconds;
+    // TODO FIXME we don't have response yet!
```

Comment:
Remove commented code ?

Reply:
Those are parts that we should include if we add stateful models support. I will remove TODO